src/FunFair.CodeAnalysis.Tests/FunFair.CodeAnalysis.Tests.csproj

```diff
@@ -18,7 +18,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.9.8" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.4.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
```

Comment:
Also needs a reference to Microsoft.VisualStudio.Threading.Analyzer with the Private thing that the other ones do